### PR TITLE
DOC: signal.freqz: fix example

### LINKED ISSUE #21510: DOC: signal.freqz: problem with `fs`
### Issue with current documentation:

Example
```py
from scipy import signal
import numpy as np
b = signal.firwin(80, 0.5, window=('kaiser', 8))
w, h = signal.freqz(b)
```
The default fs for firwin is 2.But the default fs for freqz is 2*pi.

image

So the cutoff frequency in this picture is false.

### Idea or request for content:

I'd like to submit a pr to fix it.😁

### Additional context (e.g. screenshots, GIFs)

_No response_

### PR DESCRIPTION
**Pull request by qqwqqw689**
#### Reference issue

closes #21510
#### What does this implement/fix?

change the fs for firwin.
#### Additional information

I didn't build the doc. So it may need some checks.

### CODE REVIEW

**Comment by DietBru** [comment]
> I didn't build the doc. So it may need some checks.

As described in the [Contributing to the SciPy documentation], you can check a [preview] of the rendered documentation by clicking on ``Details`` on the "**Check the rendered docs here!**" entry in the CI window below. 

A more streamlined version of the example, especially with no more cut-off text, looks like this:
```python

    >>> import numpy as np
    >>> from scipy import signal
    >>> import matplotlib.pyplot as plt
    ...
    >>> taps, f_c = 80, 1.0  # number of taps and cut-off frequency
    >>> b = signal.firwin(taps, f_c, window=('kaiser', 8), fs=2*np.pi)
    >>> w, h = signal.freqz(b)
    ...
    >>> fig, ax1 = plt.subplots(tight_layout=True)
    >>> ax1.set_title(f"Frequency Response of {taps} tap FIR Filter " +
    ...               f"($f_c={f_c}$ rad/sample)")
    >>> ax1.axvline(f_c, color='black', linestyle=':', linewidth=0.8)
    >>> ax1.plot(w, 20 * np.log10(abs(h)), 'C0')
    >>> ax1.set_ylabel("Amplitude in dB", color='C0')
    >>> ax1.set(xlabel="Frequency in rad/sample", xlim=(0, np.pi))
    ...
    >>> ax2 = ax1.twinx()
    >>> angles = np.unwrap(np.angle(h))
    >>> ax2.plot(w, angles, 'C1')
    >>> ax2.set_ylabel("Angle in radians", color='C1')
    >>> ax2.grid(True)
    >>> plt.show()
```
Otherwise, LGTM.

PS: For future Pull Requests, don't forget to consult the [PR Checklist], before submitting one.

[PR Checklist]: https://docs.scipy.org/doc/scipy/dev/contributor/development_workflow.html#checklist-before-submitting-a-pr
[Contributing to the SciPy documentation]: https://docs.scipy.org/doc/scipy/dev/contributor/rendering_documentation.html#rendering-documentation
[preview]: https://output.circle-artifacts.com/output/job/42c2f18e-d6f5-4ddf-b72c-2fa4ac76da8c/artifacts/0/html/reference/generated/scipy.signal.freqz.html

**Comment by qqwqqw689** [comment]
@DietBru thanks!

**Comment by qqwqqw689** [comment]
It looks ok now.
[link](https://output.circle-artifacts.com/output/job/dd118835-f0c2-4017-a9f0-3e5ac1cab623/artifacts/0/html/reference/generated/scipy.signal.freqz.html#scipy.signal.freqz)

File: scipy/signal/_filter_design.py [hunk 1]
```diff
@@ -366,21 +366,22 @@ def freqz(b, a=1, worN=512, whole=False, plot=None, fs=2*pi,
...
 
     >>> import matplotlib.pyplot as plt
-    >>> fig, ax1 = plt.subplots()
-    >>> ax1.set_title('Digital filter frequency response')
-
-    >>> ax1.plot(w, 20 * np.log10(abs(h)), 'b')
-    >>> ax1.set_ylabel('Amplitude [dB]', color='b')
-    >>> ax1.set_xlabel('Frequency [rad/sample]')
+    >>> fig, ax1 = plt.subplots(tight_layout=True)
+    >>> ax1.set_title(f"Frequency Response of {taps} tap FIR Filter ($f_c={f_c}$ rad/sample)")
```
Comment (DietBru) [inline scipy/signal/_filter_design.py:375]: According to the [PEP8 and SciPy] section of the [SciPy Contributor Guide], the line length limit is _88_ characters. Please convert this line into two shorter ones.

[PEP8 and SciPy]: https://docs.scipy.org/doc/scipy/dev/contributor/pep8.html#pep8-scipy
[SciPy Contributor Guide]: https://docs.scipy.org/doc/scipy/dev/contributor/contributor_toc.html

**Review by DietBru — CHANGES_REQUESTED** [review]
One small comment. Otherwise, I think it is good to go.

**Review by DietBru — APPROVED** [review]
Looks good now. Thanks, @qqwqqw689.

**Review by j-bowhay — APPROVED** [review]
Thanks for your first contribution to SciPy @qqwqqw689 and thanks @DietBru for the review

**Pull request merged**